### ISSUE TITLE
Wildcard tokens are seemingly failing to load/save when using api.autoToken

### ISSUE THREAD

**Issue by Kickbut101** [MrPrimate/tokenizer#184]
Unsure entirely if tokenizer issue. But I think I have mostly narrowed down enough variables to at least pose the question here.

Browser: Firefox 133.0.3
Foundry v12.331
Game System: PF2e v6.7.2
Plugin: v4.3.16

When I try to auto token against a token for a familiar it gives me the error in a red toast notification: 
`Failed to load token: "tokenizer/npc-images/*.png", falling back to "icons/svg/mystery-man.svg"`

For troubleshooting I have disabled all modules except for module management and Tokenizer and the issue still persists. The issue is showing up when logged in as GM, so it shouldn't be an upload issue (additionally, the directory I'm working in is the default one from tokenizer setup.config, so shouldn't be a foundry permissions issue).

See screenshot, I have the FF console export too
![image](https://github.com/user-attachments/asset
[console-export-2024-12-21_21-16-14.txt](https://github.com/user-attachments/files/18220671/console-export-2024-12-21_21-16-14.txt)
s/cb29f88a-dc93-44ee-b6a6-0a0cf7cdd28e)

**Comment by MrPrimate** [comment]
This is kind of expected, if you open Tokenizer on a wildcard NPC it won't add an image to the token panel.

You can however get around this with the following macro:

```javascript
let actor = game.actors.getName("TEST");
let tokenFilename = (await actor.getTokenDocument()).texture.src;
game.modules.get("vtta-tokenizer").api.autoToken(actor, { tokenFilename });
```

This may result in the issue of the same image having a wildcard token created multiple times. 

If the intent is to create auto tokens from all your wild card images, then a better macro would be to use something like:

```javascript
let actor = game.actors.getName("TEST");
let tokenFilenames = await actor.getTokenImages();
for (const tokenFilename of tokenFilenames) {
await game.modules.get("vtta-tokenizer").api.autoToken(actor, { tokenFilename });
}
```

**Comment by Cthoodles** [comment]
Sorry, just to clarify, does the module basically not work properly with wildcard tokens? That may explain some issues I'm having getting tokenizer to work at all with tokens provided by paizo's paid token pack modules. I have tokenizer set up for dynamic token rings, but whenever I try to edit the token of a monster provided by the paid token pack, I get this little popup and then nothing changes whatsoever

![Image](https://github.com/user-attachments/assets/a71eb618-3421-4430-98d3-5db4ddbdb58b)

**Comment by MrPrimate** [comment]
> Sorry, just to clarify, does the module basically not work properly with wildcard tokens? That may explain some issues I'm having getting tokenizer to work at all with tokens provided by paizo's paid token pack modules. I have tokenizer set up for dynamic token rings, but whenever I try to edit the token of a monster provided by the paid token pack, I get this little popup and then nothing changes whatsoever
> 
> ![Image](https://github.com/user-attachments/assets/a71eb618-3421-4430-98d3-5db4ddbdb58b)

No, it works with wildcards, it is a specific use of the API that struggles as when the API is called it doesn't have a token image to start with. Please create a separate issue for your problem detailing how I can replicate (I have the paizo packs). Thanks.
